music/src/core/recorder.ts

```diff
@@ -59,6 +63,7 @@ export class Recorder {
   private notes: NoteSequence.Note[] = [];
   private onNotes: Map<number, NoteSequence.Note>;
   private midiInputs: WebMidi.MIDIInput[] = [];
+  private startRecordAtFirstNote: boolean;
```

Comment:
argh, i thought i sent this already, but can this be "startRecordingAtFirstNote", please? It sounds a bit more human. That's my last comment, pinky swear i LGTM after